app/react/components/CodeEditor.module.css

```diff
@@ -4,6 +4,8 @@
   --text-cm-string-color: var(--red-3);
   --text-cm-number-color: var(--green-1);
   --text-cm-keyword-color: var(--ui-blue-dark-9);
+  --text-cm-comment-color: var(--orange-2);
+  --text-cm-variable-name-color: var(--green-2);
```

Comment:
use a `--ui-` color as those are the standard ones

Reply:
fixed